slave.mk

```diff
@@ -1350,8 +1354,8 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_RFS_TARGETS)) : $(TARGET_PATH)/% : \
 		export RFS_SPLIT_FIRST_STAGE=y
 		export RFS_SPLIT_LAST_STAGE=n
 
-		j2 -f env files/initramfs-tools/union-mount.j2 onie-image.conf > files/initramfs-tools/union-mount
```

Comment:
@saiarcot895 Here the j2 command use `onie-image.conf` as data source, so the env variables can't be use in j2 template now...